src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java

```diff
@@ -39,10 +41,15 @@ private VariantAnnotatorEngine(final AnnotationManager annots, final FeatureInpu
      * @param annotationsToExclude list of annotations to exclude (pass an empty list to indicate that there are no exclusions)
      * @param dbSNPInput input for variants from a known set from DbSNP or null if not provided.
      *                   The annotation engine will mark variants overlapping anything in this set using {@link htsjdk.variant.vcf.VCFConstants#DBSNP_KEY}.
+     * @param featureInputs list of inputs with known variants.
```

Comment:
done. the non-null is implied, as always when null is not explicitly mentioned (but i added this comment anyway, just for you)